ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -1703,7 +1965,214 @@ FEDataManager::restrictData(const int f_data_idx,
 
     IBTK_TIMER_STOP(t_restrict_data);
     return;
-} // restrictData
+} // restrictData_side
+
+void
+FEDataManager::restrictData_cell(const int f_data_idx,
```

Comment:
Same comment as David's above --- we should try to avoid multiple implementations of nearly identical functionality. Can we have just a single function that can switch between side- and cell-centered representations?